templates/artist/includes/artist_detail_updates.html

```diff
@@ -31,12 +19,6 @@
                                 {{ form.image.label }}:
                                 {{ form.image }}
                             </p>
-                            <h5>-- OR --</h5>
```

Comment:
Yeah, it was confusing for people. Most artist have no idea what an image URL is or how to use it so they didn't know what to enter in this spot or what it was used for